{"CVE Code": "CVE-2017-18207", "Description": "The Wave_read._read_fmt_chunk function in Lib/wave.py in Python through 3.6.4 does not ensure a nonzero channel value, which allows attackers to cause a denial of service (divide-by-zero and exception) via a crafted wav format audio file. NOTE: the vendor disputes this issue because Python applications \"need to be prepared to handle a wide variety of exceptions.", "NVD Link": "https://nvd.nist.gov/vuln/detail/CVE-2017-18207", "Vulnerability Categories": "Denial of service", "Affected Products": "\nPython\u00bbPythonVersions  up to, including, (<=) 3.6.4cpe:2.3:a:python:python:*:*:*:*:*:*:*:*Matching versions", "EPSS Score": "0.09% Probability of exploitation activity in the next 30 days\n~ 39 % Percentile, the proportion of vulnerabilities that are scored at or less", "EPSS History": "\n #  Date  Old EPSS Score  New EPSS Score  Delta (New - Old) \n 1  2024-01-09  0.09%  0.09%  +0.00 \n 2  2023-03-07  0.89%  0.09%  -0.79 ", "CVSS Scores": "4.3", "CWE": [{"CWE_ID": "CWE-369 Divide By Zero", "CWE_link": "/cwe-details/369/Divide-By-Zero.html"}], "Reference": [{"ref_link": "http://lists.opensuse.org/opensuse-security-announce/2020-01/msg00040.html", "ref_desc": "[security-announce] openSUSE-SU-2020:0086-1: important: Security updateCVEs referencing this url", "ref_cve_link": "https://www.cvedetails.com/reference-url-info/M6uRwT4u8GAnPEJg053zPAL-EMw.html"}, {"ref_link": "https://bugs.python.org/issue32056", "ref_desc": "Issue 32056: Improve exceptions in aifc, sunau and wave - Python trackerIssue Tracking;Vendor Advisory", "ref_cve_link": "N/A"}]}